docker-compose_swarm.yml

```diff
@@ -0,0 +1,46 @@
+version: '3.8'
+services:
+  postgres:
+    image: "postgres:${POSTGRES_VERSION:-17.0-bookworm}"
+    restart: always
+    volumes:
+      - $HOME/MobSF/postgresql_data:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD_FILE=/run/secrets/mobsfDB_password
+      - POSTGRES_DB=mobsf
+    networks:
+        - mobsf
+    secrets:
+      - mobsfDB_password
+  mobsf:
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD_FILE=/run/secrets/mobsfDB_password
+      - POSTGRES_DB=mobsf
+      - POSTGRES_HOST=postgres
+      - POSTGRES_PORT=5432
+      - MOBSF_API_KEY_FILE=/run/secrets/mobsf_api_key
+    image: ${MOBSF_IMAGE:-opensecurity/mobile-security-framework-mobsf:v4.0.7}
```

Comment:
can this be pointing to latest?

Reply:
I thought that it's better to refer to specific version in specific release. In this way it should be more consistent. Also it's similar to postgres(I've left corresponding comments below)

Reply:
We prefer to use the latest image for all examples. It makes sense for posrgres, but for mobsf image, please tag the latest.